shared/actions/chat2/index.js

```diff
@@ -36,6 +36,10 @@ const inboxRefresh = (
   action: Chat2Gen.InboxRefreshPayload | Chat2Gen.LeaveConversationPayload,
   state: TypedState
 ) => {
+  if (!state.config.loggedIn) {
+    return
+  }
+
```

Comment:
At the end of this function we are still [returning something](https://github.com/keybase/client/pull/13670/files#diff-62d6b6a8baeb0bb4a699ed1965bcbf00R63). There's not action creator for the event that uses this. Is the return at the end still necessary?

Same for the `unboxRows` below.

Reply:
? there's a safeTakeEveryPure